Islandora-CLAW/islandora
# Issue 1291

**GitHub Issue**: (https://github.com/Islandora-CLAW/CLAW/issues/1291)

# What does this Pull Request do?
Odd failures will occur is a user who has not been assigned the hardcoded fedoraAdmin role
attempts to add a media type that is set to save to the fedora filesystem.

# What's new?
This fix adds a filter to the controller so the user will not see media types they cannot use.

# How should this be tested?
Create a user with the Administrator role, but without fedoraAdmin role.
Log in or masquerade as that user.
Have at least one media type that saves to fedora and one that does not.
(This setting is found under the field settings for the file field in the Media type).
Navigate to the Add Media button on the Media tab of any object.
You should only see media types that do NOT save to fedora.


# Interested parties
 @Islandora-CLAW/committers
